Lib/test/test_importlib/util.py

```diff
@@ -23,9 +24,11 @@
 
 EXTENSIONS = types.SimpleNamespace()
 EXTENSIONS.path = None
+EXTENSIONS.pathlike = None
```

Comment:
These add attributes aren't really necessary as you can construct the pathlib instances on-demand.